handbook/engineering/cloud/index.md

```diff
@@ -85,10 +85,12 @@ Other:
 
 - [Keegan Carruthers-Smith](../../../company/team/index.md#keegan-carruthers-smith) will not be isolating his work to a single team. Instead, he will serially choose tasks that he thinks are important to work on and he will post updates to the most relevant tracking issue on GitHub. This is an experiment for the next month and we will evaluate the outcome on 2020-08-17. Tomás will continue to be his manager during this experiment.
 
-## Hiring status
+## Growth plan
 
-_Updated 2020-07-03_
+We've validated that while having a team focus on Cloud is very important, there continues to be a need for work to happen on other areas of our backend infrastructure. To that end, we'll split this team into two, Cloud and [Backend Infrastructure](../backend-infrastructure/index.md), as soon as we hired enough that each of those teams could have a minimum of 3 engineers, but for the time being, the Cloud team will have to handle any emerging high priority backend infrastructure goals that aren't directly related to the ultimate goal of the team.
 
-We are hiring for these roles:
+Each of these teams should not grow larger than 8 engineers. In the mean-time, Tomás will be the manager of both teams, but we're looking to hire Engineering Managers for both down the line.
```

Comment:
Are we sure we want to limit to 8 engineers, or specify engineers? Given what we discussed in cloud sync, perhaps the language could change to something like "Each of these teams should limit growth to X" - which might cover both?

Reply:
We're talking about engineers here because there's a limit to how many direct reports one engineering manager can have before getting spread to thin. 8 is an upper limit.